google-cloud-firestore/lib/google/cloud/firestore/collection.rb

```diff
@@ -0,0 +1,663 @@
+          #   # Get a collection reference
+          #   cities_col = firestore.col "cities"
+          #
+          #   # Create a query
+          #   query = cities_col.order(:name).
+          #                      select(:population).from(:cities)
+          #
+          #   # Get/run a query
+          #   firestore.get(query).each do |city|
+          #     puts "#{city.document_id} has #{city[:population]} residents."
+          #   end
+          #
+          def order field, direction = :asc
+            query.order field, direction
+          end
+
+          ##
+          # Creates a query object with method `offset` called on it. (See
+          # {Query#offset}.)
+          #
+          # @param [Integer] num The number of results to skip.
+          #
+          # @return [Query] A query with `offset` called on it.
+          #
+          # @example
+          #   require "google/cloud/firestore"
+          #
+          #   firestore = Google::Cloud::Firestore.new
+          #
+          #   # Get a collection reference
+          #   cities_col = firestore.col "cities"
+          #
+          #   # Create a query
+          #   query = cities_col.limit(5).offset(10).
+          #                      select(:population).from(:cities)
+          #
+          #   # Get/run a query
+          #   firestore.get(query).each do |city|
+          #     puts "#{city.document_id} has #{city[:population]} residents."
+          #   end
+          #
+          def offset num
+            query.offset num
+          end
+
+          ##
+          # Creates a query object with method `limit` called on it. (See
+          # {Query#limit}.)
+          #
+          # @param [Integer] num The maximum number of results to return.
+          #
+          # @return [Query] A query with `limit` called on it.
+          #
+          # @example
+          #   require "google/cloud/firestore"
+          #
+          #   firestore = Google::Cloud::Firestore.new
+          #
+          #   # Get a collection reference
+          #   cities_col = firestore.col "cities"
+          #
+          #   # Create a query
+          #   query = cities_col.offset(10).limit(5).
+          #                      select(:population).from(:cities)
+          #
+          #   # Get/run a query
+          #   firestore.get(query).each do |city|
+          #     puts "#{city.document_id} has #{city[:population]} residents."
+          #   end
+          #
+          def limit num
+            query.limit num
+          end
+
+          ##
+          # Creates a query object with method `start_at` called on it. (See
+          # {Query#start_at}.)
+          #
+          # @param [Object] values The field value to start the query at.
+          #
+          # @return [Query] A query with `start_at` called on it.
+          #
+          # @example
+          #   require "google/cloud/firestore"
+          #
+          #   firestore = Google::Cloud::Firestore.new
```

Comment:
This documentation should point out how this relates to orderBy.

Reply:
Agreed. I will update the documentation.